lint-config.js

```diff
@@ -1,79 +1,80 @@
 module.exports = {
-	"parser": "babel-eslint",
+  "parser": "babel-eslint",
 
-	"parserOptions": {
-		"sourceType": "module",
-		"ecmaFeatures": {
-			"binaryLiterals": false,
-			"blockBindings": false,
-			"defaultParams": false,
-			"forOf": false,
-			"generators": true,
-			"objectLiteralComputedProperties": false,
-			"objectLiteralDuplicateProperties": false,
-			"objectLiteralShorthandMethods": false,
-			"objectLiteralShorthandProperties": false,
-			"octalLiterals": false,
-			"regexUFlag": false,
-			"regexYFlag": false,
-			"templateStrings": false,
-			"unicodeCodePointEscapes": false,
-			"jsx": true,
-		},
-	},
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "binaryLiterals": false,
+      "blockBindings": false,
+      "defaultParams": false,
+      "forOf": false,
+      "generators": true,
+      "objectLiteralComputedProperties": false,
+      "objectLiteralDuplicateProperties": false,
+      "objectLiteralShorthandMethods": false,
+      "objectLiteralShorthandProperties": false,
+      "octalLiterals": false,
+      "regexUFlag": false,
+      "regexYFlag": false,
+      "templateStrings": false,
+      "unicodeCodePointEscapes": false,
+      "jsx": true,
+      "legacyDecorators": true,
```

Comment:
this is the only non whitespace change. Right now babel only supports transforming decorators in a legacy way that has since changed. So that's what everyone is using. Once babel supports transforming the new decorators syntax, we can remove this.

Reply:
babel-eslint@8 didn't require us to have this, but babel-eslint@9 does